composer.json

```diff
@@ -15,14 +15,15 @@
         "php-http/message": "^1.6",
         "php-http/message-factory": "^1.0",
         "phpro/soap-client": "^0.7.11",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0",
+        "symfony/http-foundation": "^3.4"
```

Comment:
The library itself does not depends from the `symfony/http-foundation` project, only one middleware that it provides does, I'd move this in `require-dev` and add to the documentation the necessity to depend from "symfony/http-foundation" when using that middleware.

Reply:
Done.